vtr_flow/scripts/python_libs/vtr/flow.py

```diff
@@ -0,0 +1,321 @@
+        
+        #Enable power analysis in VPR
+        vpr_args["power"] = True
+        #vpr_args["activity_file"] = post_ace_activity_file.name
+        vpr_args["tech_properties"] = str(power_tech_file.resolve())
+
+    #
+    # Pack/Place/Route
+    #
+    if should_run_stage(VTR_STAGE.vpr, start_stage, end_stage):
+        #Copy the input netlist for input to vpr
+        shutil.copyfile(str(next_stage_netlist), str(pre_vpr_netlist))
+        route_fixed_W = "route_chan_width" in vpr_args
+        if ("route" in vpr_args or "place" in vpr_args) and not route_fixed_W:
+            vpr_args["route_chan_width"] = 300
+            route_fixed_W = True
+            
+        if route_fixed_W:
+            #The User specified a fixed channel width
+            vtr.vpr.run(architecture_copy, circuit_copy, pre_vpr_netlist, 
+                    output_netlist=post_vpr_netlist,
+                    command_runner=command_runner, 
+                    temp_dir=temp_dir, 
+                    vpr_args=vpr_args,
+                    rr_graph_ext=rr_graph_ext)
+        else:
+            #First find minW and then re-route at a relaxed W
+            vtr.vpr.run_relax_W(architecture_copy, circuit_copy, pre_vpr_netlist, 
+                            output_netlist=post_vpr_netlist,
+                            command_runner=command_runner, 
+                            relax_W_factor=relax_W_factor,
+                            temp_dir=temp_dir, 
+                            verbosity=verbosity, 
+                            vpr_args=vpr_args)
+
+        if not lec_base_netlist:
+            lec_base_netlist = pre_vpr_netlist
+
+    #
+    # Logical Equivalence Checks (LEC)
+    #
+    if check_equivalent:
+        for file in Path(temp_dir).iterdir():
+            if "post_synthesis.blif" in str(file) :
+                gen_postsynthesis_netlist = file.name
+                break
+        vtr.abc.run_lec(lec_base_netlist, gen_postsynthesis_netlist, command_runner=command_runner, temp_dir=temp_dir)
+
+    if check_incremental_sta_consistency:
+        vtr.vpr.cmp_full_vs_incr_STA(architecture_copy, circuit_copy, pre_vpr_netlist, 
+                            command_runner=command_runner, 
+                            vpr_args=vpr_args,
+                            rr_graph_ext=rr_graph_ext,
+                            temp_dir=temp_dir
+                            )
+    
+    if(not keep_intermediate_files):
+        next_stage_netlist.unlink()
+        exts = ('.xml','.sdf','.v')
+        if not keep_result_files:
+            exts += ('.net', '.place', '.route')
+        files = []
+        for file in Path(temp_dir).iterdir():
+            if file.suffix in exts:
+                files.append(file)
+        for p in files:
+            p.unlink()
+        if power_tech_file:
+            post_ace_activity_file.unlink()
+
+def parse_vtr_flow(temp_dir, parse_config_file=None, metrics_filepath=None, verbosity=1):
```

Comment:
The old vtr rewrite also included the parse funcitonality, I didn't test this but left it in to be tested and possibly fixed later.

Reply:
I think we should remove untested code for now.  Can you move parse_vtr_flow into it's own py file and leave it out for now?